### PR TITLE
Create doee-electricbike-proj config

### PR DESCRIPTION
**Pull request by sebastianbarry**
Creating the config for a new study

- Using standard demographic survey
- Using standard multilabels
- Mode of interest is e-bike

### CODE REVIEW

**Comment by sebastianbarry** [comment]
I haven't tested yet, my e-mission-phone serve is not starting (i posted my issue in e-mission-phone team channel)

**Comment by shankari** [comment]
@sebastianbarry have you finished testing this?

**Comment by sebastianbarry** [comment]
@shankari 

Here is testing!

| Simulator Screenshot - iPhone 13 - 2024-01-30 at 09 53 32 | Simulator Screenshot - iPhone 13 - 2024-01-30 at 09 53 39 |
|---|---|
| Simulator Screenshot - iPhone 13 - 2024-01-30 at 09 55 03 | Simulator Screenshot - iPhone 13 - 2024-01-30 at 09 55 06 |

**Comment by shankari** [comment]
waiting for input from DOEE

File: configs/doee-electricbike-proj.nrel-op.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
+{
+  "version": 1,
+  "ts": 1655143472,
+  "server": {
+      "connectUrl": "https://doee-electribike-proj-openpath.nrel.gov/api/",
+      "aggregate_call_auth": "user_only"
+  },
+  "opcode": {
+      "autogen": false,
```
Comment (sebastianbarry) [inline configs/doee-electricbike-proj.nrel-op.json:9]:
```suggestion
      "autogen": true,
```

**Comment by sebastianbarry** [comment]
@shankari DOEE provided a response, they are going with autogen opcodes. They also provided admin user emails

**Comment by shankari** [comment]
@sebastianbarry can you please fill in the admin users as well per the new process that @nataliejschultz described in the team meeting?
https://github.com/e-mission/nrel-openpath-deploy-configs/pull/45

File: configs/doee-electricbike-proj.nrel-op.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      "data_uuids_columns_exclude": [],
+      "token_generate": false,
+      "token_prefix": "nrelop",
+      "map_heatmap": true,
+      "map_bubble": true,
+      "map_trip_lines": false,
+      "push_send": false,
+      "options_uuids": true,
+      "options_emails": true
+  }
```
Comment (sebastianbarry) [inline configs/doee-electricbike-proj.nrel-op.json:86]:
```suggestion
  "admin_dashboard": {
      "admin_access": [
          "laura.pimentel@dc.gov",
          "robera.tasissa@dc.gov",
          "zbaldwin@swbid.org",
          "lalbe@swbid.org"
      ]
      "overview_users": true,
      "overview_active_users": true,
      "overview_trips": true,
      "overview_signup_trends": true,
      "overview_trips_trend": true,
      "data_uuids": true,
      "data_trips": true,
      "data_trips_columns_exclude": ["data.start_loc.coordinates", "data.end_loc.coordinates"],
      "additional_trip_columns": [],
      "data_uuids_columns_exclude": [],
      "token_generate": false,
      "token_prefix": "nrelop",
      "map_heatmap": true,
      "map_bubble": true,
      "map_trip_lines": false,
      "push_send": false,
      "options_uuids": true,
      "options_emails": true
  }
```

File: configs/doee-electricbike-proj.nrel-op.json [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      "support_upload": false,
+      "trip_end_notification": false
+  },
+  "admin_dashboard": {
+      "admin_access": [
+          "laura.pimentel@dc.gov",
+          "robera.tasissa@dc.gov",
+          "zbaldwin@swbid.org",
+          "lalbe@swbid.org"
+      ]
```
Comment (sebastianbarry) [inline configs/doee-electricbike-proj.nrel-op.json:73]:
```suggestion
      ],
```

**Comment by sebastianbarry** [comment]
@shankari done and I confirmed with @nataliejschultz !

**Comment by sebastianbarry** [comment]
This PR is good to close (replaced by #67)

**Pull request closed without merging**